lib/src/scss-shared/_nx-mask-helpers.scss

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+
+ @import 'nx-colors';
+
+ @mixin background-mask($z-index) {
+  align-items: center;
+  background-color: transparentize($nx-grey-800, 0.25);
+  display: flex;
+  flex-direction: column;
```

Comment:
The properties that pertain to laying out the mask's children probably shouldn't be part of the mixin since they are specific to how the children are laid out and structured.  So for instance `display: flex`, `flex-direction`, and `align-items` probably shouldn't be in the mixin, and I'm not sure `overflow-y` is necessary at all.

Or.. we can say that a standard way of laying out children _is_ part of the mask, and adjust `.nx-submit-mask` to follow that way instead of using flex sideways as it currently does.

Reply:
> The properties that pertain to laying out the mask's children probably shouldn't be part of the mixin since they are specific to how the children are laid out and structured. So for instance `display: flex`, `flex-direction`, and `align-items` probably shouldn't be in the mixin, and I'm not sure `overflow-y` is necessary at all.
> 
> Or.. we can say that a standard way of laying out children _is_ part of the mask, and adjust `.nx-submit-mask` to follow that way instead of using flex sideways as it currently does.

I prefer that the mask has a default way of layout out its children. I have cleaned up the `flex` use here: https://github.com/sonatype/sonatype-react-shared-components/pull/330/commits/a63884e68d58a5960e387e582a14c8c27cbfb7f8